packages/vite-node/src/utils.ts

```diff
@@ -68,7 +68,13 @@ export function isInternalRequest(id: string): boolean {
   return internalRequestRegexp.test(id)
 }
 
-const prefixedBuiltins = new Set(['node:test'])
+// https://nodejs.org/api/modules.html#built-in-modules-with-mandatory-node-prefix
+const prefixedBuiltins = new Set([
+  'node:sea',
+  'node:sqlite',
+  'node:test',
+  'node:testreporters',
```

Comment:
Sorry typo :sweat_smile: 

```suggestion
  'node:test/reporters',
```